src/Tools/AI Test Toolkit/src/Logs/AITLogEntries.Page.al

```diff
@@ -0,0 +1,379 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+page 149033 "AIT Log Entries"
```

Comment:
Accordning to the UI Cop, you should set the `UsageCategory` property, If the page should not be searchagble set it to `None`

Reply:
Thanks, added to all pages.